### ISSUE TITLE
SoundManager problem in startup logs.

### ISSUE THREAD

**Issue by PhoenixRiver** [OreCruncher/DynamicSurroundings#208]
*Mod Version:* 
DynamicSurroundings-1.12.2-3.4.7.2

*Forge Version:*
1.12.2-forge1.12.2-14.23.1.2587

*Link to crash log (if applicable):*
[main/ERROR] [FML]: Unable to determine registrant mod for org.blockartistry.DynSurround.client.sound.SoundManagerReplacement@4bf5b473. This is a critical error and should be impossible
java.lang.Throwable: null
	at net.minecraftforge.fml.common.eventhandler.EventBus.register(EventBus.java:74) [EventBus.class:?]
	at org.blockartistry.DynSurround.client.sound.SoundManagerReplacement.<init>(SoundManagerReplacement.java:124) [SoundManagerReplacement.class:?]
	at net.minecraft.client.audio.SoundHandler.<init>(SoundHandler.java:63) [cho.class:?]
	at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:472) [bib.class:?]
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:377) [bib.class:?]
	at net.minecraft.client.main.Main.main(SourceFile:123) [Main.class:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_151]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_151]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_151]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_151]
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135) [launchwrapper-1.12.jar:?]
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28) [launchwrapper-1.12.jar:?]

*Description:*
Just the above error in the startup log.

**Comment by OreCruncher** [comment]
This is a strange one.  Can you try Forge version 14.23.1.2581?  It's what I am currently using and do not see an issue.

**Comment by OreCruncher** [comment]
Oh, and double check you are using the correct version of DS with the correct version of MC.  Reason I say this is because I just dropped the 1.11.2 version of DS into a MC 1.12.2 instance and got that error.

**Comment by PhoenixRiver** [comment]
Well... that was weird... I have tried to reproduce it and it doesn't happen any more - not only that I've check the log files and there is no trace of it in their either... when I originally found it I culled it from the Official Minecraft Launcher's Game Output window which shows a random selection of information (not just FML or just MC)... so I'm now mystified where that came from because it was doing it consistently at the time!

My sincere apologies for wasting your time :(

P.S. I tried Forge 2555 (as it was to hand) as well as 2587 and the version of DS was about an hour old from Curse when I downloaded it... so I presume it was the correct version on their site - I've not changed it since though so I don't know what happened to the error! Is it maybe caused by another mod overlapping initialisation with DS and not necessarily always doing it in the same order? (I'll keep an eye out for it in future and get back to you if I can pin something down (assuming it ever happens again). Again, apologies :(
